src/raft.c

```diff
@@ -308,15 +308,17 @@ static void *getClientSession(RedisRaftCtx *rr, RaftRedisCommandArray *cmds)
     unsigned long long id = cmds->client_id;
     int nokey;
 
-    void *client_session = RedisModule_DictGetC(rr->client_session_dict, &id, sizeof(id), &nokey);
+    ClientSession *client_session = RedisModule_DictGetC(rr->client_session_dict, &id, sizeof(id), &nokey);
 
     if (nokey) {
         RaftRedisCommand *c = cmds->commands[0];
         size_t cmd_len;
         const char *cmd = RedisModule_StringPtrLen(c->argv[0], &cmd_len);
 
         if (cmd_len == 5 && strncasecmp("watch", cmd, cmd_len) == 0) {
-            client_session = RedisModule_Alloc(sizeof(void *));
+            client_session = RedisModule_Alloc(sizeof(ClientSession));
+            client_session->client_id = id;
+            client_session->session_term = raft_get_current_term(rr->raft);
```

Comment:
`term` will be different on log replay, so this is something not deterministic. Does this cause a problem somewhere? Where are we going to use this variable?

Reply:
you might be right that its not needed.  I think I was originally thinking we need to compare it, then realized we didn't.